pkg/registry/core/pod/storage/eviction.go

```diff
@@ -138,46 +138,54 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		return &metav1.Status{
 			Status: metav1.StatusSuccess}, nil
 	}
-	var rtStatus *metav1.Status
 	var pdbName string
 	err = func() error {
 		pdbs, err := r.getPodDisruptionBudgets(ctx, pod)
 		if err != nil {
 			return err
 		}
 
-		if len(pdbs) > 1 {
-			rtStatus = &metav1.Status{
-				Status:  metav1.StatusFailure,
-				Message: "This pod has more than one PodDisruptionBudget, which the eviction subresource does not support.",
-				Code:    500,
-			}
-			return nil
-		}
 		if len(pdbs) == 0 {
 			return nil
 		}
 
-		pdb := &pdbs[0]
-		pdbName = pdb.Name
 		refresh := false
```

Comment:
The refresh is on a per-pdb basis, so I think this should be set inside the loop over the pdbs.